### ISSUE TITLE
Transition into TypeScript

### ISSUE THREAD

**Issue by kennythepark** [danielcolinjames/ath#35]
TypeScript brings stricter, safer environments for collaboration work. What's your thought on converting this project into TS?

**Comment by danielcolinjames** [comment]
I'm definitely not _against_ it per se, but as of right now I still haven't really got a solid grasp on the syntax so I always find TypeScript codebases just slow me down considerably 😆  but can definitely revisit TS as I feel it's something I should learn better

**Comment by kennythepark** [comment]
FWIW, I believe it's worth not only from a pure maintenance aspect, but also just the code readability and expectability since now there's a team 😎 More types, more confidence. Good news is that we don't have to transition all at once; we can incrementally introduce TS one file at a time.

**Comment by danielcolinjames** [comment]
fair enough! (sorry just seeing this) I will go through some typescript crash courses sometime soon
